pkg/controllers/pod/pod_controller.go

```diff
@@ -293,23 +354,40 @@ func (r *PodReconciler) CollectGarbage(ctx context.Context) error {
 	return nil
 }
 
+func (r *PodReconciler) getSubnetByIP(pod *v1.Pod, subnetSetUID string) (string, error) {
```

Comment:
Maybe `getSubnetByIP` -> `getSubnetByPod`? Because the input is pod, not IP..

Reply:
Good caught. Updated this for both Pod and SubnetPort, thanks!